### PR TITLE
[Finishes #88063734] Eval grid

### PR DESCRIPTION
**Pull request by Bjvanminnen**
For eval, move the origin to the center and label every other line
image

For playlab, keep origin in bottom left, but label every other line
image

### CODE REVIEW

File: apps/src/eval/eval.js [hunk 1]
```diff
@@ -106,6 +106,27 @@ Eval.init = function(config) {
       var background = document.getElementById('background');
```
Comment (bcjordan) [inline apps/src/eval/eval.js]: Could this all be pulled out into an e.g. `this.constructCoordinateGridBackground()` ? Just to keep it separated and namespace things a bit.

File: apps/src/eval/eval.js [hunk 2]
```diff
@@ -106,6 +106,27 @@ Eval.init = function(config) {
...
+      var bbox, text;
+      for (var label = origin; label <= 200; label += 100) {
+        text = document.createElementNS('http://www.w3.org/2000/svg', 'text');
+        // Position text just inside the bottom right corner.
+        text.appendChild(document.createTextNode(label));
+        svg.appendChild(text);
+        bbox = text.getBBox();
+        text.setAttribute('x', label - origin - bbox.width - 3);
+        text.setAttribute('y', CANVAS_HEIGHT - bbox.height);
+        text.setAttribute('dominant-baseline', 'hanging');
```
Comment (bcjordan) [inline apps/src/eval/eval.js]: Check this in IE9 or iOS, I remember something didn't do well with this attribute (and I think is why making dynamic resizable text on blocks was challenging)

**Comment by bcjordan** [comment]
LGTM, just a couple of questions

File: apps/src/eval/eval.js (hunk 2 above)
Comment (Bjvanminnen) [inline apps/src/eval/eval.js]: Hmmm, not aligned quite as nicely as I'd like, but probably good enough
image

File: apps/src/eval/eval.js (hunk 1 above)
Comment (Bjvanminnen) [inline apps/src/eval/eval.js]: Did this, and in doing so realized that the studio version would actually duplicate these when you hit reset. Fixed that.

**Comment by breville** [comment]
How would it look if the numbers were actually centered on the grid lines
themselves?

**Comment by Bjvanminnen** [comment]
I considered that. It would mean you would get only half of the labels on either edge. It would also make it hard to see the - for y axis labels.

**Comment by breville** [comment]
Perhaps if the text had a background color of something like
rgba(255,255,255,0.5) to dim the line behind it?  I know from rulers that
it's easier to parse when the text is centered on the line, is all :)

**Comment by Bjvanminnen** [comment]
Could probably also just get rid of the line behind it entirely
image
Would mean we miss out on labels on the edges, but that might be okay. I'll explore this option.

**Pull request merged**